GridToPager/app/src/main/java/com/google/samples/gridtopager/fragment/ImagePagerFragment.java

```diff
@@ -86,6 +86,7 @@ public void onMapSharedElements(List<String> names, Map<String, View> sharedElem
             Fragment currentFragment = (Fragment) viewPager.getAdapter()
                 .instantiateItem(viewPager, MainActivity.currentPosition);
             View view = currentFragment.getView();
+            System.out.println("view: " + view);
```

Comment:
nit: Debugging? If it should remain, I wonder if it wouldn't be better to use `Log.{d|i}` with a TAG so it's easier for developers to notice?

Reply:
Thank you for catching this! I added this for verifying a reported issue and forgot to remove it.